app/src/main/java/com/monkeyteam/chimpagne/ui/HomeScreen.kt

```diff
@@ -24,26 +25,31 @@ import androidx.compose.ui.unit.sp
 import com.monkeyteam.chimpagne.R
 import com.monkeyteam.chimpagne.ui.components.ChimpagneButton
 import com.monkeyteam.chimpagne.ui.components.ProfileIcon
-import com.monkeyteam.chimpagne.ui.components.User
 import com.monkeyteam.chimpagne.ui.navigation.NavigationActions
 import com.monkeyteam.chimpagne.ui.navigation.Route
+import com.monkeyteam.chimpagne.viewmodels.AccountViewModel
 
+@SuppressLint("StateFlowValueCalledInComposition")
```

Comment:
to suppress the warning:  val uiState by accountViewModel.uiState.collectAsState() (see MyEventsScreen for instance)